stdlib/urllib/parse.pyi

```diff
@@ -116,10 +116,10 @@ def urldefrag(url: bytes | None) -> DefragResultBytes: ...
 def urlencode(
     query: Mapping[Any, Any] | Mapping[Any, Sequence[Any]] | Sequence[tuple[Any, Any]] | Sequence[tuple[Any, Sequence[Any]]],
     doseq: bool = ...,
-    safe: AnyStr = ...,
+    safe: _Str = ...,
     encoding: str = ...,
     errors: str = ...,
-    quote_via: Callable[[str, AnyStr, str, str], str] = ...,
+    quote_via: Callable[[AnyStr, _Str, str, str], str] = ...,
```

Comment:
`AnyStr` has no meaning here, as there is only one instance. Depending on the exact composition of `query`, `quote_via()` is called with two or four arguments. If it's called with four arguments, the first argument is a `str`. I believe that using `str` is fine, until we improve `quote_via` to also accept a two argument form.